src/Dafda/Configuration/ConsumerOptions.cs

```diff
@@ -134,6 +135,14 @@ public void WithIncomingMessageFactory(IIncomingMessageFactory incomingMessageFa
             _builder.WithIncomingMessageFactory(incomingMessageFactory);
         }
 
+        /// <summary>
+        /// Filter to be applied on the consumption of each event.
```

Comment:
The consumer will evaluate whether the message matches the filter. If it does it consumes the message and dispatches it to the handler. If not it will skip the dispatcher but commits the index.

Reply:
Yeah but put that in the comment though 🤪

Reply:
Done